README.adoc

```diff
@@ -0,0 +1,15 @@
+= Integration test repo
+
+== Purpose
+The integration test repo contains a Stackable repository with services
+used for the integration tests.
+
+The following services are available:
+
+* test-service: a Shell script which just sleeps.
+
+== Contributing
+The the integration tests and the integration test repo are developed as
```

Comment:
```suggestion
The integration tests and the integration test repo are developed as
```

Reply:
Yes, the path is for NixOS. I hope it is okay to have it there.

In NixOS you cannot really do a lot without referencing a system:
```
$ ls /bin
sh
```

Reply:
Sure, no problem. I was just wondering.

I know it's a bit of work because hashes change etc. but could you add a one line comment to the shell script explaining what this line is about? 

Just "Adding XXX to path for nixos support" or something along those lines. I know this today but I'll forget in three months

Reply:
No, problem. Done and successfully tested with the integration tests.